worker/deploy/README.md

```diff
@@ -1,61 +1,163 @@
-# Worker Node deployment
+# Worker Deployment
 
-Arrebol tasks are processed by the Workers. In a typical deployment, multiple Workers are deployed together in a single virtual machine, a `Worker Node`. As each task run in an isolated docker container, the Docker engine and other dependencies must have be installed and configured in the Worker Node. Below, we detail the configuration procedure.
+This document describes the deploy of Workers. The deply process was design to ease the deployment of Workers over multiple hosts. Before explaining the details of the deployment process, the document first describe the main concepts used on the document.
 
-## Requeriments
+## **Overview**
 
-Before the configuration and installation of Worker Node dependencies, each Worker Node virtual machine should be configured to be reached via SSH (using a rsa key pair). Also, the [Ansible](https://www.ansible.com/) automation tool should be installed in the deploy `coordination host`. This host is a machine that commands that deploy. To this end, the `coordination host` should be able to access the Worker Node through SSH.
+### _Worker Node_
+
+**Worker Node** is the host where workers run. One needs to deploy at least one Worker Node for Arrebol.
+
+### _Coordination Host_
+
+The **coordination host** is a host from which scripts are executed to perfom the deployment. The coordination host must have access to Worker Nodes via **SSH (using a rsa key pair)** to perform its function.
+
+![Worker Deployment](../../imgs/wd.png)
```

Comment:
edit this figure. it has three boxes. We need only the left-most box. below the worker nodes use the following pattern: VM N/Worker Node N

Reply:
done